js/templates/modals/orderDetail/summaryTab/fulfilled.html

```diff
@@ -71,8 +70,8 @@ <h2 class="tx4 margRTn"><%= ob.polyT('orderDetail.summaryTab.fulfilled.heading')
         }) %></div>
 
         <div class="row">
-          <span><%= ob.polyT('orderDetail.summaryTab.fulfilled.transactionIdLabel') %></span> <span class="clamp3 inline"><%= cryptocurrencyDelivery.transactionID %></span>
-          <a class="clrTEm js-copyText flexNoShrink" data-content="<%= cryptocurrencyDelivery.transactionID %>" data-status-indicator=".js-transactionIdCopiedToClipboard"><%= ob.polyT('orderDetail.summaryTab.fulfilled.copyLink') %></a>
+          <span><%= ob.polyT('orderDetail.summaryTab.fulfilled.transactionIdLabel') %></span> <span class="clamp3 inline"><%= ob.transactionID %></span>
+          <a class="clrTEm js-copyText flexNoShrink" data-content="<%= ob.transactionID %>" data-status-indicator=".js-transactionIdCopiedToClipboard"><%= ob.polyT('orderDetail.summaryTab.fulfilled.copyLink') %></a>
```

Comment:
I think josh and I decided it would be better to have the unencoded form be in the copy text since the user may have some need to get the pure text.

I'm not sure I have strong feelings on this one. @jjeffryes do you have some clarity on which way to go?

Reply:
I think it would be good to have the unencoded text be copied, as a failsafe in case the user really does want the unaltered text.